src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -772,19 +706,14 @@ def _set_timeout_variables(self) -> list[str]:
     def _start(self) -> None:
         log_level = "DEBUG" if self._debug else "INFO"
         cmd = [
-            "python3",
-            "-s",
-            "-m",
-            "jupyter",
-            "labhub",
+            "jupyterhub-singleuser",
```

Comment:
What does this change do?

Reply:
Jupyter-labhub was deprecated.  https://github.com/jupyterhub/jupyterlab-hub

Since Jupyterlab 3.something (but not 3.0) it's got working-enough hub integration built in that we can just use jupyterhub-singleuser instead (it used to not set PageConfig correctly).  I've had this change pending for a long time but never merged it, and now since I'm doing a roll-up, it felt like time to do it.